jenkins-build-pack-lambda/Jenkinsfile

```diff
@@ -107,13 +72,17 @@ node ()  {
 		}
 		
 		if(configObj.service_id){	
-			config = serviceConfigdata.loadServiceMetadata(configObj.service_id,configLoader)
+			config = serviceMetadataLoader.loadServiceMetadata(configObj.service_id)
 		}else{
 			error "Service Id is not available."
 		}
 
 	}
 
+	if(!events) { error "Can't load events module"	} //Fail here
```

Comment:
Move line 82, 83 up close to loadBuildModule call above.

Reply:
cannot move the lines up, because we need the service_config to initialize the event module.